docs/user-guide/index.rst

```diff
@@ -2,8 +2,24 @@
 
 .. include:: ModelAlignment.rsts
 
+:ref:`Prerequisite Obtaining a Pre-Trained Model <prerequisite>`
+   This section provides instructions on how to download pre-trained LLMs in .nemo format. The following section will use These base LLMs for further fine-tuning and alignment. 
+
+:ref:`Model Alignment by Supervised Fine-Tuning (SFT) <sft>`
```

Comment:
sorry one last thing -- can this be updated to <model-aligner-sft> once this is updated we can merge

Reply:
@gshennvm What's the update required here?

Reply:
https://github.com/aschilling-nv/NeMo-Aligner/blob/main/docs/user-guide/SFT.rst?plain=1#L49

Reply:
right, can we change that to model-aligner-sft?

Reply:
Yeah, @aschilling-nv I don't have permission to push to the fork. Could you update when you get a chance? Thanks!